integration/single-node-slug-deploy/check.go

```diff
@@ -86,10 +86,6 @@ func main() {
 	if err != nil {
 		log.Fatalf("Couldn't get hello running: %s", err)
 	}
-	err = verifyHealthChecks(config, services)
-	if err != nil {
-		log.Fatalf("Could not get health check info from consul: %s", err)
-	}
```

Comment:
you commented out `services` but deleted this. Do you anticipate it coming back?